rom-couchdb.gemspec

```diff
@@ -18,8 +18,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'rom', '~> 1.0.0'
-  spec.add_runtime_dependency 'couchrest'
+  spec.add_runtime_dependency 'rom', '~> 2.0.0'
```

Comment:
Maybe relax this a bit to `~> 2.0`? We won't be breaking any core APIs in minor updates :)